pkg/util/storage/manager.go

```diff
@@ -43,7 +46,23 @@ func (m *manager) BlobService(ctx context.Context, resourceGroup, account string
 		SharedAccessStartTime:  &date.Time{Time: t},
 		SharedAccessExpiryTime: &date.Time{Time: t.Add(24 * time.Hour)},
 	})
+
 	if err != nil {
+		if strings.Contains(err.Error(), "TooManyRequests") { 
+				err = &api.CloudError{
+					StatusCode: http.StatusTooManyRequests,
+					CloudErrorBody: &api.CloudErrorBody{
+						Code:    api.CloudErrorCodeThrottlingLimitExceeded,
+						Message: "ThrottlingLimitExceeded",
+						Details: []api.CloudErrorBody{
+							{
+								Message: string("Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot for more details."),
```

Comment:
Can you update the link to https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits

Reply:
updated. Thanks @swiencki